### ISSUE TITLE
Error: connect ETIMEDOUT

### ISSUE THREAD

**Issue by axel7083** [containers/podman-desktop#3912]
### Bug description

Sometime on Windows platform, when I start the podman desktop, the pods does not appear.

Using the `Troubleshooting` panel, and using `ping` gave as a result 
> Error: connect ETIMEDOUT \\.\pipe\podman-machine-default

Here is the result of some commands to get more context

```
PS > podman system connection list
Name                         URI                                                          Identity                                    Default
podman-machine-default       ssh://user@127.0.0.1:53627/run/user/1000/podman/podman.sock  C:\Users\[user]\.ssh\podman-machine-default  false
podman-machine-default-root  ssh://root@127.0.0.1:53627/run/podman/podman.sock            C:\Users\[user]\.ssh\podman-machine-default  true
PS> podman machine start
Starting machine "podman-machine-default"
Error: "podman-machine-default" is already running
PS> podman info --format json
Cannot connect to Podman. Please verify your connection to the Linux system using `podman system connection list`, or try `podman machine init` and `podman machine start` to manage a new Linux VM
Error: unable to connect to Podman socket: failed to connect: dial tcp 127.0.0.1:53627: connectex: No connection could be made because the target machine actively refused it.
PS> podman machine init
Error: podman-machine-default: VM already exists
```

### Operating system

Windows 11

### Installation Method

Installer from website/GitHub releases

### Version

1.3.1

### Steps to reproduce

Not really sure.

### Work-around

The only fix seems to be rebooting my computers. I will try rebooting the WSL only next time.

**Comment by odockal** [comment]
I am seeing very similar state of podman and podman desktop today. I can add that I restarted the system before I got into this, So I propose steps to reproduce:
1. Install podman desktop (latest development build)
2. Install podman, initialize and start
3. Pull images, create containers and pods
4. Quit the podman desktop
5. Restart the system
6. Start podman desktop and verify all works correctly - podman, containers, pods, etc.

@axel7083 Does it sound right to you these steps?
